iris-mpc-store/src/s3_importer.rs

```diff
@@ -112,7 +117,8 @@ pub async fn fetch_and_parse_chunks(
     let chunks = store.list_objects().await.unwrap();
     stream::iter(chunks)
         .filter_map(|chunk| async move {
-            if chunk.ends_with(".bin") {
+            if chunk.ends_with(".csv") {
+                tracing::info!("Processing chunk: {}", chunk);
```

Comment:
same here

Reply:
I'd like to keep this one as it can help with future debugging. it's printing one by one